static/js/src/advantage/subscribe/react/components/ModalParts/ModalBody.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Col, Notification, Spinner } from "@canonical/react-components";
+
+type ModalBody = {
+  error?: React.ReactNode;
+  children: React.ReactChild;
```

Comment:
Any reason why you opted for `ReactChild` instead of `ReactNode`? It's generally best to use `React.ReactNode` for typing children. 
https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples
```suggestion
  children: React.ReactNode;
```

Reply:
I was fooled by the name! Thanks!